ci/build_wheel_libcuopt.sh

```
@@ -3,9 +3,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 set -euo pipefail
-
 source rapids-init-pip
 
+source ./ci/use_wheels_from_prs.sh
```

Shellcheck (0.11.0)

[info] Not following: ./ci/use_wheels_from_prs.sh was not specified as input (see shellcheck -x).

(SC1091)